### ISSUE TITLE
See if there are subtitles

### ISSUE THREAD

**Issue by dereineLauch** [cpiber/NebulaEnhance#17]
Hi,
I am a user who likes to have subtitles enabled, and I would like to see when I hover over the thumbnail if there are already subtitles. 
Like this:
![image](https://user-images.githubusercontent.com/84781281/158656489-439109ae-20a4-4102-91f3-bf86d4781114.png)
Is this technically possible?
By the way, I love your extension.

**Comment by cpiber** [comment]
>  Is this technically possible?

Yes. All information is distributed over the API, therefore it is accessible for me as well.

Unfortunately, subtitles are not part of the video information, they are part of the stream information. This means the information is not sent for all videos. I would have to query the API for each video by itself, that's not a good idea and would probably result in the user getting blocked.

You would probably have better luck asking the official team to add that information. I will leave this open in case they decide to add the information to the API but not the frontend.

**Comment by dereineLauch** [comment]
Ok, thanks for your time anyway.
